web/src/components/dashboard/anytime-actions/AnytimeActionDropdown.tsx

```diff
@@ -11,7 +11,7 @@ import { AnytimeActionLicenseReinstatement, AnytimeActionLink, AnytimeActionTask
 import analytics from "@/lib/utils/analytics";
 import { Autocomplete, TextField, useMediaQuery } from "@mui/material";
 import { useRouter } from "next/router";
-import { ChangeEvent, ReactElement, useState } from "react";
+import { ChangeEvent, JSX, ReactElement, useState } from "react";
```

Comment:
[dust] I'd probably campground add a `type` before ReactElement.

[dust] Any particular reason we're returning a `JSX.Element` instead of a `ReactElement` or a `ReactNode`?

Reply:
The docs suggested React.JSX.Element but ReactElement works so let's go with the simpler import